packages/flutter/lib/src/services/hardware_keyboard.dart

```diff
@@ -618,25 +618,52 @@ enum KeyDataTransitMode {
   keyDataThenRawKeyData,
 }
 
-/// The assumbled information corresponding to a native key message.
+/// The assembled information converted from a native key message.
 ///
-/// While Flutter's [KeyEvent]s are created from key messages from the native
-/// platform, every native message might result in multiple [KeyEvent]s. For
-/// example, this might happen in order to synthesize missed modifier key
-/// presses or releases.
+/// Native key messages, meaning physically pressing or releasing
```

Comment:
```suggestion
/// Native key messages, produced by physically pressing or releasing
```
Or maybe (depending on what you meant):
```suggestion
/// Native key messages, meaning physically pressed or released
```